topgg-page.htm

```diff
@@ -4,75 +4,174 @@
+                    <li><code>clean &lt;Amount:Number&gt; [...Flags]</code> - Deletes a number of messages with optional flags (see <code>help clean</code> for more).</li>
+                    <li><code>mute &lt;User:Mention/ID&gt; [Time:Duration] &lt;Reason:Text&gt;</code> - Mutes a specified Member (optional duration).</li>
+                    <li><code>unmute &lt;User:Mention/ID&gt; [Reason:Text]</code> - Unumutes a muted Member (optional reason).</li>
+                    <li><code>kick &lt;User:Mention/ID&gt; [Reason:Text]</code> - Kicks a Member from the server.</li>
+                    <li><code>ban &lt;User:Mention/ID&gt; &lt;Reason:Text&gt; [-dd &lt;Number&gt;]</code> - Bans a Member from the server. Optional deletion days flag to delete Member messages (max. 7).</li>
+                    <li><code>unban &lt;User:ID&gt; [Reason:Text]</code> - Unbans a banned User (optional reason).</li>
+                </div>
+            </div>
+    
+            <div class="dropdown">
+                <h3> Server Settings</h3>
+                <div id="bubble-list"" class="dropdown-content">
+                    <li><code>setprefix &lt;Prefix:Text|reset&gt;</code> - Changes Radeon's prefix for the server.</li>
+                    <li><code>muterole [Role:Name/Mention/ID|reset]</code> - Sends the current mute role for the server and allows for a new one to be set.</li>
+                    <li><code>modlogs &lt;SubCommand&gt; &lt;arg&gt;</code> - Modlogs config commands for managing logs (see <code>help modlogs</code> for more).</li>
+                    <li><code>automod &lt;SubCommand&gt; &lt;...args&gt;</code> - Automod config commands for managing automod triggers and events (see <code>help automod</code> for more).</li>
+                    <li><code>override &lt;SubCommand&gt; &lt;...args&gt;</code> - Channel and command override manager (see <code>help override</code> for more).</li>
+                </div>
+            </div>
+        </div>
+
+        <br><br><br><br>
+
+        <h2>About</h2>
         <p>
-            <li><code>setprefix &lt;Prefix:Text|reset&gt;</code> - Changes Radeon's prefix for the server.</li>
-            <li><code>muterole [Role:Name/Mention/ID|reset]</code> - Sends the current mute role for the server and allows for a new one to be set.</li>
-            <li><code>modlogs &lt;SubCommand&gt; &lt;arg&gt;</code> - Modlogs config commands for managing logs (see <code>help modlogs</code> for more).</li>
-            <li><code>automod &lt;SubCommand&gt; &lt;...args&gt;</code> - Automod config commands for managing automod triggers and events (see <code>help automod</code> for more).</li>
-            <li><code>override &lt;SubCommand&gt; &lt;...args&gt;</code> - Channel and command override manager (see <code>help override</code> for more).</li>
+            The idea for Radeon came about when some developers got bored and one made a lot of PRs out of boredom
         </p>
+
     </body>
 </html>
```

Comment:
might wanna add a trailing newline, as POSIX standard dictates > https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

super nitpicky but on some systems this can actually break stuff